libs/ui-lib/src/lib/custom-file-upload/custom-file-upload.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ChangeEvent, ReactNode } from 'react';
+
+import './custom-file-upload.scss';
+
+export interface CustomFileUploadProps {
+  hasErrors: boolean;
+  errorMessage: string;
+  imageURL: string;
+  handleFileUpload: (event: ChangeEvent<HTMLInputElement>) => void;
+  uploadIcon: ReactNode;
+}
+
+export const CustomFileUpload = ({
+  imageURL,
+  uploadIcon,
+  hasErrors,
+  errorMessage,
+  handleFileUpload,
+}: CustomFileUploadProps) => {
+  return (
+    <>
+      <label htmlFor="imageFile" className="custom-file-upload event-input-border">
+        {imageURL ? (
+          <img src={imageURL} style={{ width: '50%', height: '50%' }} alt="Event" />
+        ) : (
+          uploadIcon
+        )}
+      </label>
```

Comment:
```suggestion
      <label
        htmlFor="imageFile"
        style={{ boxSizing: 'border-box' }}
        className="custom-file-upload event-input-border"
      >
        {imageURL ? (
          <img
            src={imageURL}
            style={{ width: '100%', display: 'block', borderRadius: '6.5px' }}
            alt="Event"
          />
        ) : (
          <div
            style={{
              margin: '1.8em 5em',
              boxSizing: 'border-box',
              width: 'auto',
            }}
          >
            {uploadIcon}
          </div>
        )}
      </label>
```

this is to make the image upload look good. This was brought up in #59 and was supposed to be fixed in this PR.

Please remove the inline styling in my suggestion and place them in appropriate classes.